### PR TITLE
Added Feature template

### PR DESCRIPTION
**Pull request by RishiKumarRay**
Added Feature Template 
Issue #101

### CODE REVIEW

**Comment by andrewtavis** [comment]
I'd say something like  a contribution field would be better than the extra info box, as the info for the feature should be included in the description. Your thoughts on this are welcome though :)

**Comment by andrewtavis** [comment]
Can we get a checkbox at the top of the form as well for something simple like 

- [ ] I have searched open and closed issues for duplicates

**Comment by RishiKumarRay** [comment]
@andrewtavis  You're right but i have'nt seen anything in most of the templates. Can you give any example of contribution field ? Like you can make suggestions , i will commit it.

**Comment by andrewtavis** [comment]
@RishiKumarRay, see the comment above, which is also [in the file](https://github.com/scribe-org/Scribe-iOS/pull/102/files) :)

**Review by andrewtavis — COMMENTED** [review]
Thanks so much for this :) A few edits based on my comments, and this'll be ready to merge!

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    attributes:
+      label: Descriptions
+      description: A brief description of the enhancement you propose.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshot
+      description: Please add screenshots if applicable
```
[the remark this answers is not in the document]
Comment (RishiKumarRay) [reply .github/ISSUE_TEMPLATE/feature.yml]: period as in what , sorry didn't got it?

[the remark this answers is not in the document]
Comment (andrewtavis) [reply .github/ISSUE_TEMPLATE/feature.yml]: Sorry :) Needs a period after applicable`.`

**Comment by andrewtavis** [comment]
Hi @RishiKumarRay :)

In reference to [this comment](https://github.com/scribe-org/Scribe-iOS/pull/102#issuecomment-1016864156), I did a bit of research on the checkboxes for issue templates, and I think that something like this at the top would be really great:

```yml
  - type: checkboxes
    id: terms
    attributes:
      label: Terms
      description: By submitting this issue, you agree to the following:
      options:
        - label: I have searched open and closed issues for duplicates
          required: true
        - label: I agree to follow Scribe-iOS' [Code of Conduct](https://github.com/scribe-org/Scribe-iOS/blob/main/.github/CODE_OF_CONDUCT.md)
          required: true
```

Lemme know what you think :) Hopefully the link for the code of conduct can be put in the form like that 🙏

**Comment by andrewtavis** [comment]
Also, let's swift `You can Submit your Request Now.` at the end to `Thank you for your contribution!` :)

File: .github/ISSUE_TEMPLATE/feature.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrewtavis) [reply .github/ISSUE_TEMPLATE/feature.yml]: And let's have the label for this section be `Screenshots` so that they're prompted for more than one :)

**Comment by RishiKumarRay** [comment]
@andrewtavis I have added all this but am getting an error now let me fix this, In the meantime you can check if you want to add anything else

**Review by andrewtavis — APPROVED** [review]
Looks good to me! I'll merge this and do a few final changes based on what I'm seeing when it's on GitHub. We can discuss in the issue about the next ones when I'm done seeing this live 😊

**Pull request merged**